{{cookiecutter.project_folder}}/pyproject.toml

```diff
@@ -4,31 +4,33 @@ version = "{{ cookiecutter.version }}"
 description = "{{ cookiecutter.short_description }}"
 authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
 readme = "README.md"
-packages = [{include =  "{{ cookiecutter.module_name }}", from = "{{ cookiecutter.namespace }}"}]
+package-mode = false
 
 [tool.poetry.dependencies]
 python = ">={{ cookiecutter.python_version }}, <3.13"
 {%- if cookiecutter.use_jupyterlab == "y" %}
 jupyterlab = "^3.0.0"
 {%- endif %}
-pydantic = "^1.10.7"
-fastapi = "^0.112.2"
+pydantic = "^2.9.2"
+fastapi = "^0.115.2"
 typer = "^0.12.3"
+uvicorn = "^0.32.0"
```

Comment:
Do we need uvicorn? I think based on the type of project being built the user can add uvicorn later.